app/controllers/api/concerns/act_as_api_request.rb

```diff
@@ -18,6 +18,13 @@ def skip_session_storage
         # http://stackoverflow.com/a/12205114/2394842
         request.session_options[:skip] = true
       end
+
+      def render_error(status, message, _data = nil)
```

Comment:
Why here and not api_controller.rb?

Reply:
Because the devise token auth controllers don't inherit from the api controller

Reply:
That's very true